docs/getting_started.md

```diff
@@ -216,6 +231,76 @@ Here are some resources that can be used to get you started working with `AnnDat
 - [Converting directly from R to Python](https://theislab.github.io/scanpy-in-R/#converting-from-r-to-python)
 - [Homepage for scanpy tutorials](https://scanpy.readthedocs.io/en/latest/tutorials.html)
 
+
+## Special considerations for CITE-seq experiments
+
+### Filtering cells based on ADT quality control
+
+The `SingleCellExperiment` objects stored in both the `filtered.rds` and the `processed.rds` files have not been filtered based on ADT-level statistics, but both objects contain indicator variables to quickly perform this filtering.
+
+In both objects, you can see cells flagged for removal based on ADT-level QC statistics in the alternative experiment's `discard` column as calculated by [`DropletUtils::CleanTagCounts()](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html), where `TRUE` indicates the cell should be removed.
+
+```r
+# See which cells should be removed (`TRUE`) vs. kept (`FALSE`)
+# This column is in both `filtered.rds` and `processed.rds`:
+altExp(filtered_sce)$discard
+altExp(processed_sce)$discard
+```
+
+To filter cells based on this column, use the following command, for example on `filtered_sce`:
+```r
+# Filter cells based on ADT-level QC statistics
+filtered_sce <- filtered_sce[, -which(altExp(filtered_sce)$discard)]
+```
+
+In the `processed.rds` object specifically, this information is also recorded as values `"Remove"` and `"Keep"`:
+
+```r
+# See which cells should be removed vs. kept in `processed.rds`
+processed_sce$adt_scpca_filter
+```
+
+To filter cells based on this column, use the following command:
+```r
+# Filter cells based on ADT-level QC statistics
+processed_sce <- processed_sce[, which(processed_sce$adt_scpca_filter ==  "Keep")]
+```
```

Comment:
I think I would refer to this first and then say that you can also do the same by using the `discard` column. I would emphasize using this column and what's in this column more than using the `altExp` `colData`.